kudo-operator/operator.yaml

```diff
@@ -12,14 +12,17 @@ url: https://spark.apache.org/
 tasks:
   deploy:
     resources:
-      - crds.yaml
+      - spark-operator-crd.yaml
```

Comment:
shouldn't it be plural: `spark-operator-crds.yaml` because there're multiple CRDs defined in it?